Client/FeatureFlags/NimbusFlaggableFeature.swift

```diff
@@ -94,7 +94,7 @@ struct NimbusFlaggableFeature: HasNimbusSearchBar {
         let nimbusValue = nimbusLayer.checkNimbusConfigFor(featureID)
 
         switch featureID {
-        case .pocket:
+        case .pocket, .sponsoredPocket:
```

Comment:
Also, technically, this shouldn't matter, right? If Pocket's not showing up, this won't matter at all, as far as I thought. It doesn't hurt to have it in there, either way.